routes.go

```diff
@@ -2,15 +2,33 @@ package main
 
 import (
 	"budget-helper/database"
+	"budget-helper/users"
 	"fmt"
 	"log"
 	"net/http"
 )
 
-func home(db *database.Database, w http.ResponseWriter, r *http.Request) {
+func handleHome(db *database.Database, w http.ResponseWriter, r *http.Request) {
 	_, err := fmt.Fprint(w, "Welcome to Budget Helper!\n")
 	if err != nil {
-		log.Panic(err)
+		log.Fatalf("ERROR: %v\n", err)
```

Comment:
Error strings are lower case.

We already know it's an error.

Instead, write where we threw this error from. Eg:
```
log.Fatalf("handleHome: %v\n", err)
```